workspaces/git-release-manager/plugins/git-release-manager/dev/index.tsx

```diff
@@ -58,8 +68,8 @@ createDevApp()
 
         <GitReleaseManagerPage
           project={{
-            owner: 'eengervall-playground',
-            repo: 'RMaaS-semver',
+            owner: 'goodreleaser',
```

Comment:
Does `goodreleaser` reference anything in particular? 😄

Reply:
Nothing at all, it's just a mock name, I want to create in a near future others scenarios, like a badreleaser :D to improve the scenarios without correct tags or release